### DISCUSSION TITLE
NSIS merge executables

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [wailsapp/wails#2533] [Q&A]
When building with `-nsis` 2 executable files are generated (the installer and the portable version, if I understand it correctly). But the installer needs the portable version, so is there a way to merge those 2 files so the user only has to download 1 without having to figure out how it works ?

**Comment by stffabi** [comment]
Your users only need to download the installer because the installer bundles the binary.

**Reply by FrenchGithubUser** [reply]
I might be doing something wrong then. Here is the command I used to generate the installer : `wails build -platform "windows/amd64" -nsis -o "hatt-windows-x86_64.exe"` and 2 binaries are generated. The installer is lighter than the portable binary (so I guess it doesn't bundle it), and when executing the installer, there is an error saying it's missing some file.

**Reply by stffabi** [reply]
NSIS does compress itself, and go binaries compress also very well. That's why the installer is smaller.

**Reply by FrenchGithubUser** [reply]
Okay thanks, I'll have a closer look at what is going on then. Is it normal that the go binary (portable) is still generated though ?
